examples/l1-resources/federated-database-instance.ts

```diff
@@ -0,0 +1,71 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { CfnFederatedSettingsOrgRoleMapping } from 'awscdk-resources-mongodbatlas';
+
+interface AtlasStackProps {
+  readonly tenantName: string;
+  readonly projId: string;
+  readonly profile: string;
+  readonly roleId: string;
+  readonly testS3Bucket: string;
+}
+
+export class CDKFederatedDatabaseInstanceExample extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const atlasProps = this.getContextProps();
+    const federatedDatabaseInstance = new CfnFederatedSettingsOrgRoleMapping(this, 'FederatedDatabaseInstance', {
+      tenantName: atlasProps.tenantName,
+      projectId: atlasProps.projId,
```

Comment:
nit: can we keep consistency between names? any reason why in one place is called `projectId` and the other `projId`?